packages/cqrs/test/cqrs_test.dart

```diff
@@ -155,6 +155,81 @@ void main() {
         );
       });
     });
+
+    group('perform', () {
+      test(
+          "correctly serializes operation, calls client's send and"
+          ' deserializes result', () async {
+        mockClientPost(client, Response('true', 200));
+
+        final result = await cqrs.perform(
+          ExampleOperation(),
+          headers: {'X-Test': 'foobar'},
+        );
+
+        expect(result, true);
+
+        verify(
+          client.post(
+            Uri.parse('https://example.org/api/operation/ExampleOperation'),
+            body: anyNamed('body'),
+            headers: argThat(
+              isA<Map<String, String>>()
+                  .having((h) => h['X-Test'], 'X-Test', 'foobar')
+                  .having(
+                    (h) => h['Content-Type'],
+                    'Content-Type',
+                    'application/json',
+                  ),
+              named: 'headers',
+            ),
+          ),
+        ).called(1);
+      });
```

Comment:
Doesn't it complain about the missing trailing comma?

Reply:
Trailing closures in function calls do not require a trailing comma